pypeman/message.py

```diff
@@ -89,7 +89,12 @@ def add_context(self, key, msg):
 
     def to_dict(self):
         """
-        Convert the current message object to a dict. Payload is pickled.
+        Convert the current message object to a dict, that can be converted 
+            to a json.
+        Therefore the payload and all the context is pickled and base64 encoded.
+
+        Warning: the returned dict cannot be converted to json if meta contains
```

Comment:
Should we forbid non jsonable data in context ?

Reply:
Not sure what the best solution is.
Just wanted to highlight, that there might be an issue.
Documenting and forbidding non-jsonable types is a valid option.
We should just make sure, that the error message would be understandable.